packages/node-cli/src/CLI.js

```diff
@@ -69,12 +72,14 @@ module.exports = class CLI {
 			}
 
 			program
-				.version(CLI_VERSION, VERSION_OPTION)
+				.version(CLI_VERSION, VERSION_OPTION, NodeTranslationService.getMessage(VERSION_HELP))
 				.option(
 					`${INTERACTIVE_ALIAS}, ${INTERACTIVE_OPTION}`,
 					NodeTranslationService.getMessage(INTERACTIVE_OPTION_DESCRIPTION),
 					this._validateInteractive
 				)
+				.helpOption(HELP_OPTION_ALIAS_NAME, NodeTranslationService.getMessage(COMMAND_OPTIONS.HELP))
```

Comment:
We could also just use something like above for interactive option `'${HELP_OPTION_ALIAS}, ${HELP_OPTION}'`